File: Examples/Android/Aspose-Words-Cloud-SDK-for-Android/src/main/java/com/aspose/words/model/SectionLinkCollection.java
package com.aspose.words.model;

import java.util.*;
import com.aspose.words.model.Link;
public class SectionLinkCollection {
  private List<SectionLink> SectionLinkList = new ArrayList<SectionLink>();
  private Link link = null;
  /**
	 * getSectionLinkList
	 * Gets List<SectionLink>
	 * @return SectionLinkList
	 */
  public List<SectionLink> getSectionLinkList() {
    return SectionLinkList;
  }

	/**
	 * setSectionLinkList
	 * Sets List<SectionLink>
	 * @param SectionLinkList List<SectionLink>
	 */
  public void setSectionLinkList(List<SectionLink> SectionLinkList) {
    this.SectionLinkList = SectionLinkList;
  }

  /**
	 * getLink
	 * Gets Link
	 * @return link
	 */
  public Link getLink() {
    return link;
  }

	/**
	 * setLink
	 * Sets Link
	 * @param link Link
	 */
  public void setLink(Link link) {
    this.link = link;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class SectionLinkCollection {\n");
    sb.append("  SectionLinkList: ").append(SectionLinkList).append("\n");
    sb.append("  link: ").append(link).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words-Cloud-SDK-for-.NET/src/Com/Aspose/Words/Model/SectionLinkCollection.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Com.Aspose.Words.Model 
{
  /// <summary>
  /// 
  /// </summary>  
  public class SectionLinkCollection 
  {           
        /// <summary>
        /// Initializes a new instance of the <see cref="SectionLinkCollection" /> class.
        /// </summary>
        /// <param name="SectionLinkList">SectionLinkList.</param>
        /// <param name="Link">Link.</param>
        public SectionLinkCollection(List<SectionLink> SectionLinkList = default(List<SectionLink>), WordsApiLink Link = default(WordsApiLink))
        {
                        this.SectionLinkList = SectionLinkList;
                        this.Link = Link;
        }

        /// <summary>
        /// Gets or sets SectionLinkList
        /// </summary>    
        public List<SectionLink> SectionLinkList { get; set; }

        /// <summary>
        /// Gets or sets Link
        /// </summary>    
        public WordsApiLink Link { get; set; }

        /// <summary>
        /// Get the string presentation of the object
        /// </summary>
        /// <returns>String presentation of the object</returns>
        public override string ToString()  
        {
          var sb = new StringBuilder();
          sb.Append("class SectionLinkCollection {\n");
          sb.Append("  SectionLinkList: ").Append(SectionLinkList).Append("\n");
          sb.Append("  Link: ").Append(Link).Append("\n");
          sb.Append("}\n");
          return sb.ToString();
        }
    }
}
